web/src/main/java/com/graphhopper/countries/GermanySpatialRule.java

```diff
@@ -15,17 +15,18 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package com.graphhopper.routing.util.spatialrules.countries;
+package com.graphhopper.countries;
```

Comment:
As I said: in the web module it is probably wrong, and IMO it can be in the core in a separate package.